### DISCUSSION TITLE
暗色模式优化

### DISCUSSION THREAD

**Discussion by YOUGUD** [BewlyBewly/BewlyBewly#676] [Ideas]
别的新功能先不说，作者大大能不能优化一下暗色模式，像这些选中框，logo，导航栏什么的在暗色模式下彩色能不能保留下来。毕竟界面整那么好看了，有点小遗憾，如果能保留就完美了！
![微信截图_20240427034340](https://github.com/BewlyBewly/BewlyBewly/assets/165377898/43bcd6bf-2dc2-4f9d-a8e6-87f722d12898)
![微信截图_20240427034351](https://github.com/BewlyBewly/BewlyBewly/assets/165377898/55ca6cdb-689f-448f-b5d4-b5191143677b)

**Comment by hakadao** [comment]
不打算，深色模式下白色會顯得更獨特一點
